app/src/main/java/to/dev/dev_android/view/main/view/CustomWebViewClient.kt

```diff
@@ -39,8 +42,8 @@ class CustomWebViewClient(private val context: Context, private val binding: Act
             }
         }
 
-        for (i in 0 until overrideUrlList.size) {
-            if (url.contains(overrideUrlList[i])) {
+        for (element in overrideUrlList) {
+            if (url.contains(element)) {
                 return false
             }
         }
```

Comment:
This could be a good place to use Kotlin's `any()`.
```
        if (overrideUrlList.any { url.contains(element) }) return false
```

Reply:
Yeah... I shouldn't have included that change, sorry for that, I was going to create a different PR for it.

Reply:
It's not a big deal. Actually It's nice to see people apply the boy scout rule.  Maybe putting this changes in a new commit would be enough :)